src/mount/mastercomm.cc

```diff
@@ -374,6 +375,21 @@ static bool fs_threc_send_receive(threc *rec, bool filter, PacketHeader::Type ex
 const uint8_t* fs_sendandreceive(threc *rec, uint32_t expected_cmd, uint32_t *answer_leng) {
 	// this function is only for compatibility with Legacy code
 	sassert(expected_cmd <= PacketHeader::kMaxOldPacketType);
+
+#ifdef _WIN32
+    // Code specific to Windows
```

Comment:
From the #ifdef can be inferred that the code is specific for Windows, probably the comment should be related to why it is only required for Windows.